tests/test_helpers.py

```diff
@@ -12,6 +12,9 @@
     (["helper", "a_grppub_readlist", "test the lister",
       "A list to test the lister", "pdf", "--purpose", "Test the lister"],
      "test_the_lister has been added in reading_lists\n"),
+    (["helper", "a_manurev", "A. Einstein", "2020-09-15", "-c", "reject", "-r",
```

Comment:
please use the full, not the shortened versions, of the arguments as this test becomes some form of documentation and is easier to read that way.